llvm/test/Transforms/Util/flattencfg.ll

```diff
@@ -25,13 +25,13 @@ define void @test_not_crash(i32 %in_a) #0 {
 entry:
   %cmp0 = icmp eq i32 %in_a, -1
   %cmp1 = icmp ne i32 %in_a, 0
-  %cond0 = and i1 %cmp0, %cmp1
+  %cond0 = or i1 %cmp0, %cmp1
   br i1 %cond0, label %b0, label %b1
 
 b0:                                ; preds = %entry
   %cmp2 = icmp eq i32 %in_a, 0
   %cmp3 = icmp ne i32 %in_a, 1
-  %cond1 = or i1 %cmp2, %cmp3
```

Comment:
Why is this changing the original test? I would expect additional tests, and updates to the existing functions